packages/foreman/tfm/tfm.spec

```diff
@@ -71,6 +71,7 @@ Obsoletes: %{scl_prefix}rubygem-concurrent-ruby < 1.0.1-2
 Obsoletes: %{scl_prefix}rubygem-dalli < 2.6.4-4
 Obsoletes: %{scl_prefix}rubygem-ejs < 1.1.1-4
 Obsoletes: %{scl_prefix}rubygem-flot-rails < 0.0.3-8
+Obsoletes: %{scl_prefix}rubygem-faraday_middleware
```

Comment:
All others are versions so we can later re-introduce them with a newer version/release if needed.

Reply:
The majority are (https://github.com/theforeman/foreman-packaging/pull/2994/files#diff-707343da1976a0fa7839fb1086a261b9R107). What happens if you need to re-introduce the same version that you are obsoleting?

Reply:
They include a release so you can bump the release. See rubygem-flot-rails above. It's `< 0.0.3-8` so you can introduce `0.0.3-8` to replace the old `0.0.3-7` package we used to have.

Reply:
I'll follow the majority convention.